develop/dev-guide-proxysql-integration.md

```diff
@@ -0,0 +1,677 @@
+
+<SimpleTab grouId="startup-tidb">
+
+<div label="TiDB Cloud" value="tidb-cloud">
+
+It is recommended to use [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) directly when you need hosting TiDB services (for example, you cannot manage it yourself, or you need a cloud-native environment). To build a TiDB cluster in production environment, refer to [Create a TiDB cluster](https://docs.pingcap.com/tidbcloud/create-tidb-cluster).
+
+</div>
+
+<div label="Deploy Locally" value="tiup">
+
+The production environment is much more complex than the test environment. To deploy an on-premises production cluster, it is recommended to refer to [Deploy a TiDB cluster using TiUP](/production-deployment-using-tiup.md) and then deploy it based on hardware conditions.
+
+</div>
+
+</SimpleTab>
+
+## 2. Startup ProxySQL
+
+### Install by yum
+
+1. Add the repository:
+
+    ```shell
+    cat > /etc/yum.repos.d/proxysql.repo << EOF
+    [proxysql]
+    name=ProxySQL YUM repository
+    baseurl=https://repo.proxysql.com/ProxySQL/proxysql-2.4.x/centos/\$releasever
+    gpgcheck=1
+    gpgkey=https://repo.proxysql.com/ProxySQL/proxysql-2.4.x/repo_pub_key
+    EOF
+    ```
+
+2. Install ProxySQL:
+
+    ```shell
+    yum install proxysql
+    ```
+
+3. Start ProxySQL:
+
+    ```shell
+    systemctl start proxysql
+    ```
+
+### Other ways
+
+To install ProxySQL using other ways, refer to the [ProxySQL README](https://github.com/sysown/proxysql#installation) or the [ProxySQL installation documentation](https://proxysql.com/documentation/).
+
+## 3. Configure ProxySQL
+
+To use ProxySQL as a proxy for TiDB, you need to configure ProxySQL. The required configuration items are listed in the following sections. For more details about other configuration items, refer to the [ProxySQL official documentation](https://proxysql.com/documentation/).
+
+### Simple introduction
+
+ProxySQL uses a port to manage configuration, which is **_ProxySQL Admin interface_**, and a port to proxy, which is **_ProxySQL MySQL Interface_**.
+
+- **_ProxySQL Admin interface_**: To connect to the admin interface, you can use a `admin` user to read and write configuration, or use a `stats` user to read part of statistics (cannot read or write configuration). The default credentials are `admin:admin` and `stats:stats`. For security reasons, you can use the default credentials to connect locally, but to connect remotely, you need to configure a new user, which is often named `radmin`.
+- **_ProxySQL MySQL Interface_**: Used as a proxy to forward SQL to the configured service.
+
+![proxysql config flow](/media/develop/proxysql_config_flow.png)
+
+There are three layers in ProxySQL configuration: `runtime`, `memory`, and `disk`. You can change the configuration of the `memory` layer only. After modifying the configuration, you can use `LOAD xxx TO runtime` to make the configuration effective, and/or you can use `SAVE xxx TO DISK` to save to the disk to prevent configuration loss.
```

Comment:
```suggestion
There are three layers in ProxySQL configurations: `runtime`, `memory`, and `disk`. You can change the configuration of the `memory` layer only. After modifying the configuration, you can use `LOAD xxx TO runtime` to make the configuration effective, and/or you can use `SAVE xxx TO DISK` to save to the disk to prevent configuration loss.
```